staging/com.smartnews/index.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+const Tp = require("thingpedia");
+const API_URL = "https://039ev0y88l.execute-api.us-west-1.amazonaws.com/test/v2";
+const USER = "jack";  // use 1 for now
+//const cheerio = require("cheerio");
+
+module.exports = class SmartNewsDevice extends Tp.BaseDevice {
+    constructor(engine, state) {
+         super(engine, state);
+
+         this.uniqueId = "com.smartnews";
+         this.name = "SmartNews";
+         this.description = "SmartNews latest articles";
```

Comment:
You should move the device name and description to the `#_[name]` and `#_[description]` annotations of the class in the manifest (so they can be translated in the future)